testing_openstack_code_changes_with_operators.md

```diff
@@ -124,3 +124,47 @@ Status:
       /var/lib/kolla/config_files/config.json from config (ro,path="neutron-api-config.json")
       /var/run/secrets/kubernetes.io/serviceaccount from kube-api-access-rhk5m (ro)
 ```
+
+# Testing OpenStack code changes in the CI
+
+With [recent change](https://github.com/openstack-k8s-operators/ci-framework/pull/1892)
+in the ci-framework it is now possible to run operator's CI jobs together with
+the depends-on patch proposed for the upstream OpenStack project in upstream
+gerrit.
+
+This require to replace `openstack-k8s-operators-content-provider` job with
+`openstack-meta-content-provider` job as dependency for other jobs in
```

Comment:
Can you split description into two parts? One part that describes variables required by openstack-meta-content-provider and the second about variables for other jobs with a dependency to `openstack-meta-content-provider`

Reply:
Done